common/environment.go

```diff
@@ -35,10 +35,26 @@ var VisibleEnvironmentVariables = []EnvironmentVariable{
 	EEnvironmentVariable.ShowPerfStates(),
 	EEnvironmentVariable.PacePageBlobs(),
 	EEnvironmentVariable.DefaultServiceApiVersion(),
+	EEnvironmentVariable.ClientSecret(),
+	EEnvironmentVariable.CertificatePassword(),
 }
 
 var EEnvironmentVariable = EnvironmentVariable{}
 
+func (EnvironmentVariable) ClientSecret() EnvironmentVariable {
```

Comment:
Is there any existing standard for what these env vars are named?  (e.g. the AWS ones, below, follow a naming convention, IIRC).  If there's no existing standard, how about prefixing them with something like AZCOPY_SPA_. Thoughts?